package.json

```diff
@@ -45,7 +45,6 @@
     "eslintrc": "^1.0.6",
     "express": "4.16.0",
     "express-actuator": "^1.1.0",
-    "express-jwt": "5.3.0",
     "express-jwt-permissions": "1.2.1",
```

Comment:
Since `express-jwt` is being removed, is `express-jwt-permissions`  removed as well?

Reply:
nope, despite the name similarity, the latter is not dependent on the former, and we are using it.